test-harness/modules/mocker/endpoints/marathon.py

```diff
@@ -372,22 +375,33 @@ class MarathonEndpoint(RecordingTcpIpEndpoint):
     """An endpoint that mimics DC/OS root Marathon"""
     def __init__(self, port, ip=''):
         super().__init__(port, ip, MarathonHTTPRequestHandler)
-        self._context.data["endpoint-content"] = {"apps": [NGINX_TASK_ALWAYSTHERE, ]}
-        self._context.data["leader-content"] = {"leader": "127.0.0.2:80"}
+        self._reset()
 
     def reset(self, *_):
         """Reset the endpoint to the default/initial state."""
         with self._context.lock:
             super().reset()
-            self._context.data["endpoint-content"] = {"apps": [NGINX_TASK_ALWAYSTHERE, ]}
-            self._context.data["leader-content"] = {"leader": "127.0.0.2:80"}
+            self._reset()
 
-    def enable_nginx_task(self, *_):
```

Comment:
Why app instead of task? I am OK with both, but I just wonder what is reasoning behind it.